.github/workflows/codeql.yaml

```diff
@@ -46,18 +46,12 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      run: |
+        cd src
+        dotnet build
 
-    # ?? Command-line programs to run using the OS shell.
-    # ?? https://git.io/JvXDl
-
-    # ?? If the Autobuild fails above, remove it and uncomment the following three lines
-    #    and modify them (or add more) to build your code if your project
-    #    uses a compiled language
-
-    #- run: |
-    #   make bootstrap
-    #   make release
+    #  uses: github/codeql-action/autobuild@v1
```

Comment:
Should this uses line be commented out?

Reply:
I updated the comments in that section to make it clearer.

autobuild doesn't build correctly (this just started happening), so we do a manual build